libraries/Matter/examples/MatterColorLight/MatterColorLight.ino

```diff
@@ -35,7 +35,12 @@ const uint8_t ledPin = 2;  // Set your pin here if your board has not defined LE
 #endif
 
 // set your board USER BUTTON pin here
-const uint8_t buttonPin = 0;  // Set your pin here. Using BOOT Button. C6/C3 use GPIO9.
+#ifdef BOOT_PIN
+const uint8_t buttonPin = BOOT_PIN;  // Set your pin here. Using BOOT Button.
```

Comment:
BOOT_PIN is always defined. Leave just this line and it will be enough for someone to change the pin to something else. Same for the other examples.

Reply:
done.